transform/mattermost-analytics/models/staging/mm_telemetry_prod/stg_mm_telemetry_prod__performance_events.sql

```diff
@@ -0,0 +1,73 @@
+{{
+    config({
+        "tags":"hourly",
+    })
+}}
+
+WITH MM_TELEMETRY_PROD AS 
+(SELECT _dbt_source_relation	  AS _dbt_source_relation	
```

Comment:
`_dbt_source_relation` is a column that is added by `dbt_utils.union_relations`. Table `events` doesn't include that.